beaker-vue/src/router/index.ts

```diff
@@ -45,7 +45,11 @@ const defaultRouteMap: Routes = {
     "admin": {
       "path": "/admin",
       "component": () => import('@/pages/BeakerAdmin.vue'),
-    }
+    },
+    "playground": {
+      "path": "/playground",
```

Comment:
left a recording of this theme/pelette/guideline page as a reply to a review comment.